guidelines.md

```diff
@@ -256,8 +256,14 @@ Examples of other design patterns:
 - WebSocket: TBA
 - gRPC: TBA
 
-## Project organization
-Separate specifications from implementation code. OpenAPI files **SHOULD** be in **TBD**
+## Development Workflow Recommendations
+
+### Contract-First Development
+We **encouraged** to follow a Contract-First workflow:
```

Comment:
Pirking, men "encourage" antar jag? Kanske "We encourage you to follow a Contract-First workflow:"